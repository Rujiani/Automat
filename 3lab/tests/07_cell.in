vu√¥ng pos;
pos = *_*;
pos;