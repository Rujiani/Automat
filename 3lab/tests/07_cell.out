(0, 0) free
